### DISCUSSION TITLE
Add to docs screenshot of all endpoints after installation, how it will looks like.

### DISCUSSION THREAD

**Discussion by david-shiko** [fastapi-users/fastapi-users#468] [Ideas]
Please provide docs image how it will looks like after installation will be done. I found only this one, but it's not seems to be docs of `fastapi-users`
https://i.ytimg.com/vi/FCSL2d6uhVs/maxresdefault.jpg
![image](https://user-images.githubusercontent.com/27439358/106753298-8c215e00-663c-11eb-8394-0326c82bdc6f.png)

**Comment by frankie567** [comment]
Thank you for the suggestion, I'll consider it!
